### PR TITLE
[v2.2] Add Changelog entry for Envoy % escaping patch

### PR DESCRIPTION
**Pull request by aidanhahn**
Signed-off-by: Aidan Hahn <aidanhahn@datawire.io>

### CODE REVIEW

File: OPENSOURCE.md [hunk 1]
```diff
@@ -191,4 +191,4 @@ libraries:
     urllib3              1.26.5     MIT license
     webencodings         0.5.1      3-clause BSD license
     websocket-client     0.57.0     3-clause BSD license
-    zipp                 3.6.0      MIT license
+    zipp                 3.7.0      MIT license
```
Comment (LukeShu) [inline OPENSOURCE.md:194]: What's with this change?

**Comment by LukeShu** [comment]
The envoy bump to include the patch isn't landed yet, is it?

File: docs/releaseNotes.yml [hunk 2]
```diff
@@ -46,6 +46,18 @@ items:
         - title: 3906
           link: https://github.com/emissary-ingress/emissary/issues/3906
         docs: https://www.getambassador.io/docs/edge-stack/latest/topics/running/statistics/8877-metrics/
+      - title: Envoy configuration % escaping
+        type: feature
+        body: >-
+          Emissary is now leveraging a new Envoy Proxy patch that allows Envoy to accept escaped
+          '%' characters in its configuration. This means that error_response_overrides and other
+          custom user content can now contain '%' symbols escaped as '%%'
```
Comment (LukeShu) [inline docs/releaseNotes.yml:54]: I'm fine with this wording.

**Review by AliceProxy — APPROVED** [review]

**Review by kflynn — DISMISSED** [review]
Hit it!

**Review by AliceProxy — APPROVED** [review]

**Pull request merged**